_episodes/05-history.md

```diff
@@ -348,6 +348,30 @@ moving backward and forward in time becomes much easier.
 > 4. `$ git checkout <unique ID of last commit> data_cruncher.py`
 >
 > 5. Both 2 and 4
+>
+> > ## Solution
+> >
+> > Lets look at each command to see what will happen.
+> > 
+> > 1. `$ git checkout HEAD`
+> >
+> > When Jennifer runs this command, git will not change any of her files because this command does not specify which files she wants to checkout.
```

Comment:
When no file names are specified, `git checkout <ref>` moves the HEAD pointer to the specified <ref>. Therefore, `git checkout HEAD` does not change any files not because there are no files specified but because `git checkout` behaves differently: it moves `HEAD` to `HEAD`. `git checkout HEAD~1`, for example, will change contents of all files. I'd suggest either expanding this explanation or adding a small section mentioning that `checkout` (and `reset`) behave differently when file names are specified on the command line.